.gitignore

```diff
@@ -1,7 +1,8 @@
-.github
-package-lock.json
-node_modules
 .stylelintrc.json
 .hintrc
```

Comment:
[Required] Remove this `.stylelintrc.json` and `.hintrc` files from gitignore, they are very important to your project (they are linters setup files)